wot/ModelReader.py

```diff
@@ -320,7 +322,7 @@ def readNormal(self, data, IS_NEW):
 				z = -float(pkz&0x7f)/0x7f
 			else:
 				z = float(pkz^0x7f)/0x7f
-			return (x, y, z)
+			return (x, z, y)
```

Comment:
Are you sure about is? I'm asking because it seemed like it works in this order for some models.